realsense_navigation/package.xml

```diff
@@ -1,23 +1,21 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
   <name>realsense_navigation</name>
   <version>1.1.0</version>
   <description>RealSense Camera navigation package allowing use of Intel 3D cameras with the navigation stack</description>
 
-  <maintainer email="amit.moran@intel.com">Amit Moran</maintainer>
   <maintainer email="matthew.c.curfman@intel.com">Matt Curfman</maintainer>
+  <maintainer email="mark.d.horn@intel.com">Mark Horn</maintainer>
 
   <license>BSD 3-clause. See license attached</license>
 
   <url type="website">http://www.ros.org/wiki/RealSense</url>
 
   <author email="amit.moran@intel.com">Amit Moran</author>
-  <author email="matthew.c.curfman@intel.com">Matt Curfman</author>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>rospy</build_depend>
- 
-  <run_depend>roscpp</run_depend>
-  <run_depend>rospy</run_depend>
+
+  <exec_depend>roslaunch</exec_depend>
+  <exec_depend>realsense_camera</exec_depend>
+  <exec_depend>rgbd_launch</exec_depend>
```

Comment:
Yes, we should replace rgbd_launch with packages we are directly referencing in our launch files; my mistake as I was modeling after the [ros_astra_launch](https://github.com/orbbec/ros_astra_launch) example which only used 'rgbd_launch'.
We seem to have a long list:
`turtlebot_bringup, turtlebot_navigation, map_server, rviz, rosbag, amcl, gmapping`
Does this seem to be complete?